polls/views.py

```diff
@@ -58,7 +60,7 @@ class AdminDashboardView(LoginRequiredMixin, UserPassesTestMixin, View):
     template_name = "polls/admin_dashboard.html"
 
     def test_func(self):
-        return self.request.user.is_superuser
+        return not self.request.user.is_superuser
```

Comment:
Isn't this view only for admin users? Why do we need to let other users have access to it?

Reply:
I forgot to resolve it after testing, thanks for the observation.